src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -169,7 +169,7 @@ public static void TestDispose()
             Assert.Equal(IntPtr.Zero, chain.ChainContext);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
This test is not using RSA+SHA1, it's using DSA(+SHA1).

Reply:
The chain fails to build.
```
<test name="System.Security.Cryptography.X509Certificates.Tests.ChainTests.TestResetMethod" type="System.Security.Cryptography.X509Certificates.Tests.ChainTests" method="TestResetMethod" time="0.0028312" result="Fail">
<failure exception-type="Xunit.Sdk.TrueException">
<message>
Chain built validly\nExpected: True\nActual: False
</message>
<stack-trace>
at System.Security.Cryptography.X509Certificates.Tests.ChainTests.TestResetMethod() in /home/tester/runtime/src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs:line 194 at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor) at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr) in /home/tester/runtime/src/libraries/System.Private.CoreLib/src/System/Reflection/MethodInvoker.cs:line 69
</stack-trace>
</failure>
</test>
```

Reply:
OK... but.... why?

If the same setting in RHEL is disabling DSA+SHA1, then we have a reason, but then we should make a second SignatureSupport for SupportsDsaSha1Signatures.  And then we shouldn't turn off the test, we should expect false and a particular set of flags.

It's quite possible this is the only test that's building chains with DSA, if it's the only place failing.

Reply:
Looked a bit closer:
```
DSA.Create().SignData(Array.Empty<byte>(), HashAlgorithmName.SHA1)
```
doesn't throw `CryptographicException`. So DSA+SHA1 signatures seem to be supported.

For this test, `chain.ChainStatus[0].Status` is `NotSignatureValid`.